<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BLR13-1011163KW" failures="18" tests="34" name="RestapiProject" time="26.265" errors="0" timestamp="10 Apr 2022 05:57:26 GMT">
  <testcase classname="TestCases.AllEmployeeData" name="checkContentLenght" time="0.009"/>
  <testcase classname="TestCases.AllEmployeeData" name="checkContentType" time="0.002"/>
  <testcase classname="TestCases.AllEmployeeData" name="checkCookies" time="0.001"/>
  <testcase classname="TestCases.AllEmployeeData" name="checkResponseTime" time="0.003"/>
  <testcase classname="TestCases.AllEmployeeData" name="checkResposeBody" time="0.027"/>
  <testcase classname="TestCases.AllEmployeeData" name="checkStatusCode" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at TestCases.AllEmployeeData.checkStatusCode(AllEmployeeData.java:59)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="TestCases.AllEmployeeData" name="checkcontentEncoding" time="0.002"/>
  <testcase classname="TestCases.AllEmployeeData" name="checkserverType" time="0.004">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at TestCases.AllEmployeeData.checkserverType(AllEmployeeData.java:109)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="TestCases.AllEmployeeData" name="checkstatusLine" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at TestCases.AllEmployeeData.checkstatusLine(AllEmployeeData.java:86)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase classname="TestCases.SingleEmployeeData" name="checkContentLenght" time="0.002"/>
  <testcase classname="TestCases.SingleEmployeeData" name="checkContentType" time="0.001"/>
  <testcase classname="TestCases.SingleEmployeeData" name="checkResponseTime" time="0.001"/>
  <testcase classname="TestCases.SingleEmployeeData" name="checkResposeBody" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.SingleEmployeeData.checkResposeBody(SingleEmployeeData.java:46)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="TestCases.SingleEmployeeData" name="checkStatusCode" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at TestCases.SingleEmployeeData.checkStatusCode(SingleEmployeeData.java:53)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="TestCases.SingleEmployeeData" name="checkserverType" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at TestCases.SingleEmployeeData.checkserverType(SingleEmployeeData.java:83)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="TestCases.SingleEmployeeData" name="checkstatusLine" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at TestCases.SingleEmployeeData.checkstatusLine(SingleEmployeeData.java:68)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase classname="TestCases.PostEmployeeRecord" name="checkContentType" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at TestCases.PostEmployeeRecord.checkContentType(PostEmployeeRecord.java:95)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase classname="TestCases.PostEmployeeRecord" name="checkResposeBody" time="0.002"/>
  <testcase classname="TestCases.PostEmployeeRecord" name="checkStatusCode" time="0.001"/>
  <testcase classname="TestCases.PostEmployeeRecord" name="checkcontentEncoding" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [gzip] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at TestCases.PostEmployeeRecord.checkcontentEncoding(PostEmployeeRecord.java:109)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkcontentEncoding -->
  <testcase classname="TestCases.PostEmployeeRecord" name="checkserverType" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at TestCases.PostEmployeeRecord.checkserverType(PostEmployeeRecord.java:102)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="TestCases.PostEmployeeRecord" name="checkstatusLine" time="0.001"/>
  <testcase classname="TestCases.PutRecord" name="checkContentType" time="0.001"/>
  <testcase classname="TestCases.PutRecord" name="checkResposeBody" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.PutRecord.checkResposeBody(PutRecord.java:69)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="TestCases.PutRecord" name="checkStatusCode" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at TestCases.PutRecord.checkStatusCode(PutRecord.java:78)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase classname="TestCases.PutRecord" name="checkcontentEncoding" time="0.001"/>
  <testcase classname="TestCases.PutRecord" name="checkserverType" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at TestCases.PutRecord.checkserverType(PutRecord.java:100)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="TestCases.PutRecord" name="checkstatusLine" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at TestCases.PutRecord.checkstatusLine(PutRecord.java:85)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase classname="TestCases.DeleteEmployeeRecord" name="checkContentType" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at TestCases.DeleteEmployeeRecord.checkContentType(DeleteEmployeeRecord.java:78)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase classname="TestCases.DeleteEmployeeRecord" name="checkResposeBody" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestCases.DeleteEmployeeRecord.checkResposeBody(DeleteEmployeeRecord.java:55)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase classname="TestCases.DeleteEmployeeRecord" name="checkStatusCode" time="0.001"/>
  <testcase classname="TestCases.DeleteEmployeeRecord" name="checkcontentEncoding" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [gzip] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at TestCases.DeleteEmployeeRecord.checkcontentEncoding(DeleteEmployeeRecord.java:92)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkcontentEncoding -->
  <testcase classname="TestCases.DeleteEmployeeRecord" name="checkserverType" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx]
at TestCases.DeleteEmployeeRecord.checkserverType(DeleteEmployeeRecord.java:85)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase classname="TestCases.DeleteEmployeeRecord" name="checkstatusLine" time="0.0"/>
</testsuite> <!-- RestapiProject -->
